crates/proposal-signing-backends/src/dkg.rs

```diff
@@ -190,3 +165,11 @@ fn webb_proposals_typed_chain_converter(
         webb_proposals::TypedChainId::Ink(id) => TypedChainId::Ink(id),
     }
 }
+
+pub fn make_execute_proposal_key(data_hash: [u8; 32]) -> [u8; 64] {
+    let mut result = [0u8; 64];
+    let prefix = b"execute_proposal_with_signature_";
+    result[0..32].copy_from_slice(prefix);
+    result[32..64].copy_from_slice(&data_hash);
+    result
+}
```

Comment:
Same as above, we could better name this function:
```suggestion
pub fn make_ acknowledge_proposal_key(data_hash: [u8; 32]) -> [u8; 64] {
    let mut result = [0u8; 64];
    // make sure that this byte array is 32 bytes.
    let prefix = b"acknowledge_proposal_fixed_key_";
    result[0..32].copy_from_slice(prefix);
    result[32..64].copy_from_slice(&data_hash);
    result
}
```